superset-frontend/src/components/Table/index.tsx

```diff
@@ -259,6 +265,7 @@ export function Table<RecordType extends object>(
     recordCount,
     onRow,
     allowHTML = false,
+    childrenColumnName,
```

Comment:
@justinpark `childrenColumnName` is not a direct property of the table. It belongs to the `expandable` configuration:

```
<Table 
   expandable={
      childrenColumnName: 'children',
   }
>
```